src/StoreProvider.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import StoreContext from '~/store/StoreContext';
+
+import { UI, GlobalSearch } from './store';
+import globalSearchAPI from '~/api/globalSearch';
+
+const ui = new UI();
+const globalSearch = new GlobalSearch(ui, globalSearchAPI);
```

Comment:
Momentan haben wir nur einen Store für das UI (führt momentan nur Sprachzustand) und einen für die globale Suche.
Hier werden weitere Abhängigkeiten als Konstruktorparameter mitgegeben. In diesem Beispiel wird der ui-Store in den globalSearch-Store gegeben, damit für die globale Suche der Sprachcode abgefragt werden kann.